### ISSUE TITLE
[Request] Recently Played options & AddOn grid views(?)

### ISSUE THREAD

**Issue by sondregronas** [chrisbevan/skin.pellucid#42]
First of all this is by far my favorite skin for Kodi, but there are some features I would love to see. I've only been using Kodi for a couple of days so its probable that this is possible to be fixed by myself..

Is it possible to flip the New Movies / Continue Watching in the settings? Alternatively choose to hide them separately or use 2 rows? Currently I'm only finding a setting to hide recently watched, which also hides the Continue Watching section.

Also in the AddOns, again I'm not sure if this is possible in the skin or if this is due to how the AddOns themselves work but, is it possible to get a dashboard view in AddOns like YouTube, Vimeo, instead of a list view?

Thank you!

**Comment by sondregronas** [comment]
Also - is there a possibility to add "continue to watch TV shows" items in the Continue Watching tab? Right now all my unwatched items show up in the New videos, but would like to only access the ones I'm currently watching

**Comment by chrisbevan** [comment]
Hi,

- Flipping the new / recently watched: Not something I'll make an option for, but you can do this yourself if you're happy to modify the skin yourself. Let me know and I'll post the relevant file / changes. The downside to that is that your changes will be overwritten when I post updates.

- Addon dashboard Grid view. I'll look into it. Should be possible. 

- Continue to watch tv shows: There should already be an option for this ("Continue Watching"). Does it not appear? Are you watching your content via a plugin / plex or from a local library?

**Comment by sondregronas** [comment]
Hey and thanks for your reply.

I'm watching my content with the PlexKodiConnect AddOn (external repository) which afaik should only create shortcuts to the local library.

I do not find anything on Continue Watching in any of the menus. In the Recently Added / Continue Watching section the Continue tab is empty unless I've stopped a show the mid-episode.

When I was looking through skins I did see a functioning "Continue Watching Shows" menu.

Is there supposed to be a new section for unwatched episodes, or would they appear in Continue Watching?

I just tried resetting the Skin settings without luck.
